includes/ucf-section-common.php

```diff
@@ -173,7 +173,100 @@ public static function format_shortcode_output( $content ) {
 			$rep = preg_replace( "/(<p>)?\[\/($block)](<\/p>|<br \/>)?/", "[/$2]", $rep );
 			return $rep;
 		}
+
+		/**
+		 * Called by the wp_head filter
+		 * Determines if a section has a stylesheet to print
+		 * into the head of the document
+		 * @author Jim Barnes
+		 * @since 1.0.1
+		 **/
+		public static function get_section_stylesheets() {
+			global $post;
+
+			wp_enqueue_style( 'section-styles', plugins_url( 'static/css/empty.css', UCF_SECTION__PLUGIN_FILE ) );
```

Comment:
Consider prefixing style and script names with 'ucf', e.g. 'ucf-section-styles'

Reply:
Alternatively, consider removing these enqueued scripts and manually echo-ing each inline style/script in the head using `wp_head` and `wp_footer` hooks instead of with `wp_add_inline_style()`/`wp_add_inline_script()` (this would help avoid forcing the user to download two empty files on pageload)